charts/radar-s3-connector/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "7.6.0-hotfix"
 description: A Helm chart for RADAR-base s3 connector. This connector uses Confluent s3 connector with a custom data transformers. These configurations enable a sink connector. See full list of properties here https://docs.confluent.io/kafka-connect-s3-sink/current/configuration_options.html#s3-configuration-options
 name: radar-s3-connector
-version: 0.3.2
+version: 0.4.0
```

Comment:
Minor version increases are better used bigger updates

Reply:
I have put the version at 0.3.3.